sensor/temperature.cpp

```diff
@@ -0,0 +1,16 @@
+#include <iostream>
+#include "temperature.h"
+
+using namespace std;
+
+vector<int> generateTemperatureValues(int teperatureMaxValue , int teperatureMinValue, int numberOfReadings) {

```

Comment:
Please correct the variable name for more readablity(teperatureMaxValue and teperatureMinValue)

Reply:
changes done.